java/client/src/main/java/glide/api/models/commands/scan/ScanOptions.java

```diff
@@ -28,6 +29,13 @@ public class ScanOptions extends BaseScanOptions {
      */
     private final ObjectType type;
 
+    /**
+     * If set to True, the scan will perform even if some slots are not covered by any node. It's
```

Comment:
```suggestion
     * If set to true, the scan will perform even if some slots are not covered by any node. It's
```

Reply:
Very much a nitpick, just some grammar things:
- "If set to true, the scan will proceed/be performed/be executed even if..."
- "However, it is important to note that in this mode, the scan cannot guarantee coverage of all keys in the cluster and it loses the ability to validate its progress"

Up to you if you want to adopt any of these.